### ISSUE TITLE
HydroDyn linearization

### ISSUE THREAD

**Issue by RBergua** [OpenFAST/openfast#2023]
**Bug description**
I compiled the HydroDyn driver from OpenFAST dev commit [e39453a](https://github.com/OpenFAST/openfast/commit/e39453af79726c5cb23dd6e2b2389da6b6dc2960).

I tried to run one of the r-test for the HydroDyn module: https://github.com/OpenFAST/r-test/tree/main/modules/hydrodyn/hd_5MW_OC4Semi_WSt_WavesWN

However, it seems that the Linearize line cannot be interpreted:
![image](https://github.com/OpenFAST/openfast/assets/33709140/a6ba7766-99a4-477a-9669-f7e18e64e881)

For reference, the driver input file from the r-test reads as follows:
![image](https://github.com/OpenFAST/openfast/assets/33709140/1b479f8b-def8-4fcb-8777-e59ac6fea874)

Is the linearization capability available in the HydroDyn driver? Some months ago I was working with an OpenFAST branch developed by @ebranlard and though that, maybe, that development was merged into the current OpenFAST.

**Comment by jjonkman** [comment]
What does the echo file show if you enable `Echo`?

**Comment by RBergua** [comment]
The echo file stops at the line for linearize:
![image](https://github.com/OpenFAST/openfast/assets/33709140/c2e138ea-f436-4b5b-b16b-ff52f3772dce)

**Comment by andrew-platt** [comment]
@RBergua, the `dev` branch has an additional input in the driver file: https://github.com/OpenFAST/r-test/blob/dev/modules/hydrodyn/hd_5MW_OC4Semi_WSt_WavesWN/hd_driver.inp

**Comment by RBergua** [comment]
Ops! I just saw that. There is a line for SeaState in the dev that I was missing. My bad... Let me try again.

**Comment by RBergua** [comment]
I confirm that I could make work the r-test example in dev.
![image](https://github.com/OpenFAST/openfast/assets/33709140/103cd203-3c5c-44c8-ab7d-b7e6c1905dfc)

I will try to run it in my system now. Thanks!
